ffmpeg/lpms_ffmpeg.c

```diff
@@ -114,6 +116,13 @@ int lpms_rtmp2hls(char *listen, char *outf, char *ts_tmpl, char* seg_time)
     ist = ic->streams[pkt.stream_index];
     ost = oc->streams[stream_map[pkt.stream_index]];
     int64_t dts_next = pkt.dts, dts_prev = prev_ts[pkt.stream_index];
+    if (oc->streams[pkt.stream_index]->codecpar->codec_type == AVMEDIA_TYPE_VIDEO) {
+        if (AV_NOPTS_VALUE == dts_prev) {
+          if (pkt.flags & AV_PKT_FLAG_KEY) got_video_kf = 1;
+          else goto r2hloop_end; // XXX accommodate audio-only streams
```

Comment:
Is this additional goto needed? Seems like if we do not find a video keyframe in the check on L121 execution would continue to the check on L125 and then evaluate the goto there to jump to `r2hloop_end`?

Reply:
Good catch, fixed.